core/src/main/java/com/orange/ouds/core/theme/OudsColorScheme.kt

```diff
@@ -0,0 +1,1121 @@
+        val onStatusAccentMuted: Color,
+        val onStatusAccentMutedOnBgEmphasized: Color,
+        val onStatusInfoEmphasized: Color,
+        val onStatusInfoMuted: Color,
+        val onStatusInfoMutedOnBgEmphasized: Color,
+        val onStatusNegativeEmphasized: Color,
+        val onStatusNegativeMuted: Color,
+        val onStatusNegativeMutedOnBgEmphasized: Color,
+        val onStatusPositiveEmphasized: Color,
+        val onStatusPositiveMuted: Color,
+        val onStatusPositiveMutedOnBgEmphasized: Color,
+        val onStatusWarningEmphasized: Color,
+        val onStatusWarningMuted: Color,
+        val onStatusWarningMutedOnBgEmphasized: Color,
+        val statusInfo: Color,
+        val statusNegative: Color,
+        val statusPositive: Color,
+        val statusWarning: Color,
+    )
+
+    data class Elevation(
+        val drag: Color,
+        val dragOnBgEmphasized: Color,
+        val dragOnBgSecondary: Color,
+        val modal: Color,
+        val overlayDefault: Color,
+        val overlayDefaultOnBgEmphasized: Color,
+        val overlayDefaultOnBgSecondary: Color,
+        val overlayEmphasized: Color,
+        val overlayEmphasizedOnBgEmphasized: Color,
+        val overlayEmphasizedOnBgSecondary: Color,
+        val raised: Color,
+        val raisedOnBgEmphasized: Color,
+        val raisedOnBgSecondary: Color,
+    )
+
+    data class Gradient(
+        val skeletonMiddle: Color,
+        val skeletonMiddleOnBgEmphasized: Color,
+        val skeletonStartEnd: Color,
+        val skeletonStartEndOnBgEmphasized: Color,
+    )
+
+    data class Decorative(
+        val accent1Default: Color,
+        val accent1Emphasized: Color,
+        val accent1Muted: Color,
+        val accent2Default: Color,
+        val accent2Emphasized: Color,
+        val accent2Muted: Color,
+        val accent3Default: Color,
+        val accent3Emphasized: Color,
+        val accent3Muted: Color,
+        val accent4Default: Color,
+        val accent4Emphasized: Color,
+        val accent4Muted: Color,
+        val accent5Default: Color,
+        val accent5Emphasized: Color,
+        val accent5Muted: Color,
+        val brandPrimary: Color,
+        val brandSecondary: Color,
+        val brandTertiary: Color,
+        val neutralDefault: Color,
+        val neutralEmphasized: Color,
+        val neutralMuted: Color,
+        val skinTint100: Color,
+        val skinTint200: Color,
+        val skinTint300: Color,
+        val skinTint400: Color,
+        val skinTint500: Color,
+        val skinTint600: Color,
+        val skinTint700: Color,
+        val skinTint800: Color,
+        val skinTint900: Color,
+    )
+}
+
+val OudsColorSemanticTokens.lightColorScheme: OudsColorScheme
+    get() = OudsColorScheme(
+        globalColors = OudsColorScheme.Global(
+            transparentDefault = globalColorTokens.transparentDefaultLight,
+        ),
+        actionColors = OudsColorScheme.Action(
```

Comment:
We can use `with(actionColorTokens)` to simplify the following lines.

Same comment for other parameters.